schedule/yam/agama_separate_boot.yaml

```diff
@@ -0,0 +1,11 @@
+---
```

Comment:
same for this file name, yam/whole_disk_and_boot_unattended.yaml

Reply:
Changed.